Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+Dir[File.join(__dir__, 'gemfile.d', '*.rb')].each do |bundle|
+  eval_gemfile(bundle)
```

Comment:
It is a shame dependabot doesn't understand this, where it did understand the previous format.

Reply:
in this very particular case, we could just write `eval_gemfile('gemfile.d/test.rb')` and Dependabot would work, but otherwise I agree.